src/Modules/SimplCommerce.Module.Reviews/wwwroot/admin/review-replies.module.js

```diff
@@ -0,0 +1,18 @@
+﻿/*global angular*/
+(function () {
+    'use strict';
+
+    angular
+        .module('simplAdmin.review-replies', [])
```

Comment:
I think we should use the simplAdmin.reviews instead of creating a new angular module